src/autoloads/appearance.gd

```diff
@@ -2,6 +2,8 @@ extends Node
 
 onready var Resources = get_node("/root/Resources")
 
+signal applyConfig # Apply config for menu character
```

Comment:
I'd rename this signal to something that specifies that the character customization has been accepted in the appearance screen.

Reply:
I'll change this in the other PR